content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -252,6 +252,12 @@ kubectl get csr/john -o yaml
 
 The certificate value is in Base64-encoded format under `status.certificate`.
 
+Export the issued certificate from the CertificateSigningRequest.
+
+```
+kubectl get csr john -o jsonpath='{.status.certificate}'| base64 -d > certificate.crt
+```
```

Comment:
Is `certificate.crt` correct if the example follows as:

```
This example assumes the key and certificate files are located at "/home/vagrant/work/".
First, you need to add new credentials:

kubectl config set-credentials john --client-key=/home/vagrant/work/john.key --client-certificate=/home/vagrant/work/john.crt --embed-certs=true
``` 

/cc @kubernetes/sig-security-pr-reviews

Reply:
Some concerns regarding this (_to this PR to add missing cert export step_)
1) Why use `/home/vagrant/work/` as opposed to assuming local working directory and why use `john` as opposed to more generic name.  Not necessarily a goal of mine to change those original decisions.  But I'm open to it if reviewers think that should be changed to something else in all locations, not just the line this PR adds.
2) Exporting certificate should happen right after you [get the cert](https://deploy-preview-25735--kubernetes-io-master-staging.netlify.app/docs/reference/access-authn-authz/certificate-signing-requests/#get-the-certificate), as opposed to further down when [adding to kubectl ](https://deploy-preview-25735--kubernetes-io-master-staging.netlify.app/docs/reference/access-authn-authz/certificate-signing-requests/#add-to-kubeconfig).  Either locations seems OK to me, but I'm open to suggestions.

Reply:
> 1. Why use `/home/vagrant/work/` as opposed to assuming local working directory

I agree with this

> why use john as opposed to more generic name

switching to `myuser` or something would make sense to me as well